docs/website/docs/development/architecture/podman-limitations.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Podman limitations
+sidebar_position: 40
+---
+
+The `odo dev` command is able to work either on Podman or on a Kubernetes cluster. 
+
+The motivation behind the support for the Podman platform is to lower the learning curve
+for developers working on containerized applications. As a matter of fact, Podman is simpler
+to apprehend, install and maintain than a Kubernetes cluster.
```

Comment:
It would be good to also mention that with Podman you have everything locally with much lower overhead (resources)